using_images/xpaas_images/eap.adoc

```diff
@@ -9,7 +9,120 @@
+JBoss EAP instance inside the JBoss EAP xPaaS image] is different from the process you may be used to for a regular release of JBoss EAP.
+
+=== Unsupported Configurations
+
+The following is a list of unsupported configurations specific to the JBoss EAP xPaaS image:
+
+* Using MySQL in a scaled environment with XA distributed transactions is not recommended. For applications that support both scaling and XA distributed transactions, PostgreSQL is recommended instead.
+// This is based on https://issues.jboss.org/browse/CLOUD-56
+
+== Installing the JBoss EAP xPaaS Image Streams and Application Templates
+
+To use the Red Hat xPaaS middleware images in your OpenShift project, you must first install the image streams and Source-to-Image (S2I) application templates.
+
+. Clone the Application Template repository:
++
+----
+# git clone https://github.com/jboss-openshift/application-templates
+----
+. Create the image streams and S2I templates:
++
+----
+# oc create -f application-templates/jboss-image-streams.json -n openshift
+# oc create -f application-templates/eap/ *.json -n openshift
+----
+
+You will now have the JBoss EAP xPaaS images streams and S2I application templates available to use in your OpenShift project.
+
+[[Making-Configuration-Changes-EAP]]
+== Running and Configuring the JBoss EAP xPaaS Image
+
+You can make changes to the JBoss EAP configuration in the JBoss EAP xPaaS image in multiple ways:
+
+=== Using the JBoss EAP xPaaS Image Source-to-Image (S2I) Process
+
+The recommended method to run and configure the OpenShift JBoss EAP xPaaS image is to use the OpenShift S2I process together with the application template parameters and environment variables.
+
+The S2I process for the JBoss EAP xPaaS image works as follows:
+
+. If there is a *_pom.xml_* file in the source repository, a Maven build is triggered with the contents of `*$MAVEN_ARGS*` environment variable.
++
+By default the `package` goal is used with the `openshift` profile, including the system properties for skipping tests (`*-DskipTests*`) and enabling the Red Hat GA repository (`*-Dcom.redhat.xpaas.repo.redhatga*`).
++
+The results of a successful Maven build are copied to *_EAP_HOME/standalone/deployments_*. By default, this includes:
++
+* All JAR, WAR, and EAR files from the *_target_* directory in source repository.
+* All JAR, WAR, and EAR files from the directory within the source repository specified by `*$ARTIFACT_DIR*` environment variable.
+. Any JAR, WAR, and EAR in the *_deployments_* source repository directory are copied to the *_EAP_HOME/standalone/deployments_* directory.
+. All files in the *_configuration_* source repository directory are copied to *_EAP_HOME/standalone/configuration_*.
++
+[NOTE]
+If you want to use a custom JBoss EAP configuration file, it should be named *_standalone-openshift.xml_*.
+. All files in the *_modules_* source repository directory are copied to *_EAP_HOME/modules_*.
+
+
+\**LIST OF PARAMETERS/VARIABLES**
```

Comment:
Kevin will be sending us the list of env variables, and when he does, I will include them.